src/display/screen.cpp

```diff
@@ -1586,8 +1495,8 @@ void Screen::closeHelpView()
 void Screen::showDetailView()
 {
     int y = 1;
-    ITelemetry * event;
-    StackTrace * eventTrace;
+    ITelemetry event;
+    StackTrace eventTrace;
```

Comment:
This is unnecessarily making copies of `event` and `eventTrace` later when you assign with just `=`.  Perhaps use leave it as a pointer or use a `const` reference?